src/components/Navbar/index.tsx

```diff
@@ -26,54 +28,68 @@ const Navbar: React.FC = () => {
-                            <Button type="button" onClick={toggleDropdown} className="text-white focus:outline-none">
-                                <div className="flex items-center space-x-2">
-                                    <ProfileIcon firstName={firstName} lastName={lastName} size={50} />
-                                    <span> {firstName}</span>
-                                    <DownArrowIcon />
-                                </div>
-                            </Button>
-                        ) : (
-                            <Link
-                                className="flex items-center space-x-2 bg-white text-black px-4 py-2 rounded-md hover:bg-gray-100 cursor-pointer"
-                                href={TINY_API_GOOGLE_LOGIN}
-                                data-testid="google-login"
-                            >
-                                <GoogleIcon />
-
-                                <span>Sign In</span>
-                            </Link>
-                        )}
-                    </li>
-                    <ul className={`${menuOpen ? 'block' : 'hidden'} absolute top-[8vh] right-0 bg-gray-800 p-2 z-10`}>
-                        <li>
-                            <Link href="#" className="text-white hover:bg-gray-700 block px-4 py-2">
-                                Dashboard
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href={TINY_API_LOGOUT} className="text-white hover:bg-gray-700 block px-4 py-2">
-                                Sign Out
-                            </Link>
+                    <ul className={'lg:flex space-x-4'}>
+                        <li className="relative group">
+                            {isLoggedIn ? (
+                                <Button
+                                    type="button"
+                                    onClick={() => setMenuOpen(!menuOpen)}
+                                    className="text-white focus:outline-none"
+                                >
+                                    <div className="flex items-center space-x-2">
+                                        <ProfileIcon firstName={firstName} lastName={lastName} size={50} />
+                                        <span> {firstName}</span>
+                                        <DownArrowIcon />
+                                    </div>
+                                </Button>
+                            ) : (
+                                <Button
+                                    className="flex items-center space-x-2  text-white px-4 py-2 rounded-md cursor-pointer hover:bg-gray-700"
+                                    data-testid="google-login"
+                                    onClick={() => setShowLoginModal(true)}
+                                >
+                                    <span>Sign In</span>
+                                </Button>
```

Comment:
Nit: try to avoid div and use article or section or any symantic tags

Reply:
Is it acceptable to use "article" or "section" in the navigation bar? Is this considered a good practice?